Mathlib/CategoryTheory/Monoidal/Mod_.lean

```diff
@@ -129,3 +128,35 @@ def comap {A B : Mon_ C} (f : A ⟶ B) : Mod_ B ⥤ Mod_ A where
 -- Lots more could be said about `comap`, e.g. how it interacts with
 -- identities, compositions, and equalities of monoid object morphisms.
 end Mod_
+
+section Mod_Class_
+
+open CategoryTheory Mon_Class MonoidalCategory
+
+variable (M : C) [Mon_Class M]
+
+/--
+An action of a monoid object `M` on an object `S` is the data of map
+`smul : M ⊗ S ⟶ S` that satisfies "the right commutative diagrams" -/
+class Mod_Class_ (S : C) where
+  /-- The action map -/
+  smul : M ⊗ S ⟶ S
+  /-- The action map is compatible with the monoid's multiplication -/
+  mul_smul (S) : (μ ▷ S) ≫ smul = (α_ M M S).hom ≫ (M ◁ smul) ≫ smul := by aesop_cat
+  /-- The identity acts trivially -/
+  one_smul (S) : (η ▷ S) ≫ smul = (λ_ S).hom := by aesop_cat
+
+attribute [reassoc (attr := simp)] Mod_Class_.mul_smul Mod_Class_.one_smul
+
+namespace Mod_Class_
+
+@[inherit_doc] notation "γ" => Mod_Class_.smul
+
+-- See note [reducible non instances]
+/-- The action of a monoid object on itself. -/
+abbrev regular : Mod_Class_ M M where
```

Comment:
```suggestion
abbrev leftMul : Mod_Class_ M M where
```
no?

Reply:
You mean [`Mul.toSMul`](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Algebra/Group/Action/Defs.html#Mul.toSMul)

Reply:
Okay, let's leave as if for now